packages/js/src/shared-admin/components/premium-upsell-card.js

```diff
@@ -72,9 +76,13 @@ export const PremiumUpsellCard = ( { link, linkProps, promotions } ) => {
 				className="yst-flex yst-justify-center yst-gap-2 yst-mt-4 focus:yst-ring-offset-primary-500"
 				{ ...linkProps }
 			>
-				<span>{ isBlackFriday ? __( "Claim your 30% off now!", "wordpress-seo" ) : getPremium }</span>
+				<span>{ isBlackFriday ? __( "Claim your 30% off now!", "wordpress-seo" ) : explorePremium }</span>
 				<ArrowNarrowRightIcon className="yst-w-4 yst-h-4 yst-icon-rtl" />
 			</Button>
+			<p className="yst-text-center yst-text-[11px] yst-leading[19px] yst-mt-2">
+				{ __( "Only $/€/£99 per year (ex VAT) 30-day money back guarantee - No questions asked.", "wordpress-seo" ) }
+			</p>
+			<hr className="yst-border-t-[1px] yst-border-purple-300 yst-my-4" />
```

Comment:
The default width should be `1px` already. So I don't think this `-[1px]` is needed?

Also, the `purple` should be `primary`. I'm pretty sure we want to use our brand color at least.

```suggestion
			<hr className="yst-border-t yst-border-primary-300 yst-my-4" />
```